### ISSUE TITLE
Use of apostrophes breaks the script

### ISSUE THREAD

**Issue by armaninux** [nickjj/notes#8]
Normal use of apostrophes in the note text (as in "let's, don't, etc.) breaks the script, it won't finalize. Apostrophes need to be escaped in order to work but I wonder if it just happens on my system? (Linux EndeavourOS, i3wm, kitty terminal)

**Comment by nickjj** [comment]
- If I do `notes "Let's see if this works"` it works.
- If I do `notes Let's see if this works` it fails but this is your shell letting you know you have unmatched quotes.

**Comment by linux478** [comment]
If you use a backslash to escape the apostrophe than the command should work.

**Comment by nickjj** [comment]
Yep, this is normal shell behavior, closing this as either solution posted is how you'd handle this scenario for quotes and other shell specific characters that could be part of your notes string.
